CMakeLists.txt

```
@@ -335,6 +342,20 @@ if (USE_GNUTLS)
 		target_link_libraries(datachannel PRIVATE Nettle::Nettle)
 		target_link_libraries(datachannel-static PRIVATE Nettle::Nettle)
 	endif()
+elseif(USE_MBEDTLS)
+	if(USE_SYSTEM_MBEDTLS)
+		find_package(MbedTLS REQUIRED)
```

You must set a minimum version in the `find_package`. Replace the `3` by right minimum version if needed.
```suggestion
		find_package(MbedTLS 3 REQUIRED)
```